src/chapters/thymeleaf-views/template-fragments.rst

```diff
@@ -0,0 +1,211 @@
+Template Fragments
+====================
+
+Just like methods in Java provide us with the ability to reuse useful code,
+Thymeleaf allows us to do something similar with HTML.
+
+.. index:: ! fragments
+
+**Fragments** are blocks of HTML elements that we want to use across multiple
+templates. The fragments are stored in a separate file, and they can be
+accessed by using the keywords ``fragment`` and ``replace``.
+
+A summary of these two keywords it given below, followed by another
+:ref:`video walkthrough <coding-events-fragments-vid>` to give you some more
+live-coding practice.
+
+Fragments DRY Your Code
+------------------------
+
+Anythyme you find yourself typing identical code into different templates,
+little alarm bells should go of in your head, and you should think, *How can I
+streamline this?*
```

Comment:
"Anytime you find yourself typing identical code in different templates, you may find yourself thinking of a way to streamline this."

Reply:
Fixed. Thanks.